src/wkt/tests/message_with_f32.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[cfg(test)]
+mod test {
+    use serde_json::json;
+    type Result = std::result::Result<(), Box<dyn std::error::Error>>;
+    use test_case::test_case;
+
+    #[serde_with::serde_as]
+    #[serde_with::skip_serializing_none]
+    #[derive(Clone, Debug, Default, PartialEq, serde::Deserialize, serde::Serialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct MessageWithF32 {
+        #[serde_as(as = "google_cloud_wkt::internal::F32")]
+        pub singular: f32,
+        #[serde_as(as = "Option<google_cloud_wkt::internal::F32>")]
+        pub optional: Option<f32>,
+        #[serde_as(as = "Vec<google_cloud_wkt::internal::F32>")]
+        pub repeated: Vec<f32>,
```

Comment:
Please include a `HashMap<String, f32>`, I think those appear in the generated code too.

Reply:
Done.